src/client/fontengine.cpp

```diff
@@ -211,20 +176,17 @@ unsigned int FontEngine::getDefaultFontSize()
 /******************************************************************************/
 void FontEngine::readSettings()
 {
-#if USE_FREETYPE
```

Comment:
@SmallJoker, by removing this `#ifdef` you broke possibility to build minetest without freetype and gettext:

Right now I'm trying to build the very minimal configuration and I've set `ENABLE_FREETYPE=OFF` and `ENABLE_GETTEXT=OFF` and I see the following error:

```
minetest\src\client\fontengine.cpp(184): error C3861: 'gettext': identifier not found
```

Reply:
Please explain further. This preprocessor checks against `USE_FREETYPE`, not `USE_GETTEXT`.
There's a dummy function available for `gettext()` if gettext is not available:
https://github.com/minetest/minetest/blob/008b80fe1ca9d46610b8971e2ac92fb56da8e69f/src/gettext.h#L38

Reply:
fbc8724

Reply:
Thanks a lot, this fixes the build!